webui/react/src/components/UPlot/UPlotChart/closestPointPlugin.ts

```diff
@@ -13,6 +14,7 @@ interface Point {
 }
 
 interface Props {
+  checkpointsDict?: CheckpointsDict;
   diamond?: boolean;
```

Comment:
diamond should be removed here so we can catch a handful of other places where it's still sent